packages/nextjs/app/page.tsx

```diff
@@ -5,6 +5,8 @@ import { EcosystemGrants } from "./_components/EcosystemGrants";
 import { GrantsStats } from "./_components/GrantsStats";
 import { HomepageHero } from "./_components/HomepageHero";
 
+export const revalidate = 21600; // 6 hours
+
 const Home = () => {
```

Comment:
Kind of not related to this PR but also added [`revalidate`](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#revalidate) as we discussed https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/54#issuecomment-1973525921. 

This will re static generate Home page after every 6 hours, to test you could add `console.log("Home is being re-validated!")` and update `revalidate = 30;` (30 seconds) and then `yarn next build && yern next start`

Reply:
For reference, I had to do:

```
cd packages/nextjs
yarn run build
yarn run serve
```

Your command was giving me some weird NextJS errors:
![image](https://github.com/BuidlGuidl/grants.buidlguidl.com/assets/2486142/03c2a756-1284-461d-a4aa-78d31b5e3433)

Reply:
Hmm strange lol it works fine for me : 

![Screenshot 2024-03-05 at 11 43 13 PM](https://github.com/BuidlGuidl/grants.buidlguidl.com/assets/80153681/72850feb-dc7b-4c1b-bf2c-30448692e2f9)